### PR TITLE
add diagnostics file when new monitoring instance is created

### PR DESCRIPTION
**Pull request by soujay**
This PR is a fix for [Monitoring instance "Failed to write startup diagnostics"   #2021](https://github.com/Particular/ServiceControl/issues/2021)

### CODE REVIEW

File: src/ServiceControl.Monitoring/Bootstrapper.cs [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ﻿namespace ServiceControl.Monitoring
 {
     using System;
+    using System.Configuration;
```
Comment (WilliamBZA) [inline src/ServiceControl.Monitoring/Bootstrapper.cs:4]: Why is this needed?

Comment (soujay) [reply src/ServiceControl.Monitoring/Bootstrapper.cs]: removed test code

**Review by WilliamBZA — APPROVED** [review]
